packages/cli/src/lib/config.ts

```diff
@@ -38,6 +39,14 @@ async function createConfig(): Promise<Config> {
 }
 
 export async function loadOrCreateConfig(path = DEFAULT_CONFIG_PATH): Promise<Config> {
+  if (!existsSync(path)) {
+    // no new named config file found
+    if (existsSync(join(homedir(), '/.llama-pack/llama-pack-config.json'))) {
+      // old named config file found - rename it and it's folder
+      promises.rename((join(homedir(), '/.llama-pack/llama-pack-config.json')), path);
```

Comment:
What we're doing here is a bit odd:

1. Rename/move `~/.llama-pack/llama-pack-config.json` to `~/.bubblewrap-config/bubblewrap-config.json`.
2. Rename `~/.llama-pack` to `~/.bubblewrap-config`.

So in step #2, are we not overwriting the `.bubblewrap-config` directory with the now empty `.llama-pack-config` directory?